packages/generator/src/renderer.ts

```diff
@@ -37,6 +37,7 @@ import {
   getPascalCaseName,
   formatForTsDoc
 } from "./templateHelpers";
+import { addNamespace } from "./commonTemplateHelper";
```

Comment:
Why is this in its own file and not with the other template helpers?

Reply:
I think @monkey-jeff answered this elsewhere, but, the other helpers are being moved out to the raml-toolkit, this is something that is not likely going to be reused by other SDKs. It also provides a way for other consumers to understand how the framework can be extended.